src/modules/systemlib/airspeed.h

```diff
@@ -48,6 +47,29 @@
 
 __BEGIN_DECLS
 
+enum AIRSPEED_SENSOR_MODEL {
+	AIRSPEED_SENSOR_MODEL_MEMBRANE = 0,
+	AIRSPEED_SENSOR_MODEL_SDP3X,
```

Comment:
First step would be to add `uint32 device_id	# unique device ID for the sensor that does not change between power cycles` and then decode that struct to extract the sensor type ID. But I'll do that separately to not clutter this PR.